intern/templates/pages/base.html

```diff
@@ -80,69 +80,71 @@ <h5 style = " word-wrap: break-word;">Xin chào </h5>
             </div>  
             
         </div>
+        <!-- Nav -->
         <div class="row" style ="margin-top:0px;">
             <div class="col-sm-2"></div>
         
             <div class="col-sm-8">
-                
+                <!-- Leagues -->
                 <ul id="menu">
                     <li>
                         <a href="/blog">
                         Trang chủ 
                         </a>
                     </li>
+                    <!-- nav EU football leagues -->
                     <li>
                         <a href="javascript:void(0)">
                         Giải Châu Âu
                         <span class="arrow arrow-down"></span>
                         </a>
                         <ul class="dropdown_menu">
                             <li>
-                                <a href="javascript:void(0)">Anh</a>
+                                <a href="/blog/premier-league">Anh</a>
```

Comment:
Is it possible to retrieve these URLs dynamically? In case you want to change this to `/blog/premier-league` you don't have to replace it in all templated manually.